include/autoppl/algorithm/mh.hpp

```diff
@@ -84,7 +84,7 @@ inline void mh_posterior__(ModelType& model,
             using state_t = typename util::var_traits<var_t>::state_t;
 
             if (var.get_state() == state_t::parameter) {
-                auto curr = var.get_value();
+                auto curr = var.get_value(0);
```

Comment:
Just to be clear, this is because we're still assuming parameters are univariate right?

Reply:
Yes. For any parameter, the first vector element is used for the value.